src/app/components/console/consoleview.cpp

```diff
@@ -52,23 +54,104 @@ ConsoleView::~ConsoleView()
     delete ui;
 }
 
-void ConsoleView::execute(QString pCommand)
+void ConsoleView::execute(QString &pCommand)
 {
     ui->plainTextEdit->putData("\n");
     mProcess->write(pCommand.toUtf8());
     if(mProcess->atEnd() && !dirPrinted)
     {
-        qDebug() << "dir print from exec";
         slotPrintWorkingDir();
     }
+    QProcess* console;
+    if(OsInfo()=="windows")
+    {
+        console = new QProcess(this);
+        QByteArray resultConsole;
+        QStringList args;
+        //QString cdPathAndCommand = "cd"+consolePath()+"&call "+pCommand;
+        args << "/c"  << pCommand; // << "\r" //Two commands in CMD Link : http://stackoverflow.com/questions/9888806/run-two-commands-in-one-windows-cmd-line-one-command-is-set-command
+        //need set mPath
+        //cmd->setWorkingDirectory(mPath);
+        console->start("C:\\Windows\\System32\\cmd", args);
+        //cmd->startDetached("C:\\Program Files\\Git\\bin"); //???need arguments for Git
```

Comment:
???need arguments for Git
thats look like your temp code
remove it
or use //TODO:
find example in code